learn/templates/learn/automated_list_page.html

```diff
@@ -29,26 +31,12 @@ <h1>{{ page.title }}</h1>
 </div>
 <section id="learn-root" class="great great-container">
     <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds-from-desktop">
+        <div class="govuk-grid-column-two-thirds-from-desktop govuk-!-margin-bottom-6">
             {% for learning_module in page.get_children.specific.live %}
                 {% with high_level_completion_progress|get_item:learning_module.id as module_completion_data %}
-                    <div class="great-card great-card--undecorated great-card--no-active-jump great-bg-white govuk-!-margin-bottom-4">
-                        <a href="{% pageurl learning_module %}">
-                            <div class="great-display-flex great-flex-wrap great-flex-column-until-tablet great-gap govuk-!-padding-4">
-                                <div class="great-flex-2">
-                                    <h3 class="great-card__link great-title-link great-card__link--heading govuk-heading-m govuk-!-margin-top-2 govuk-!-margin-bottom-2">
-                                        {{ learning_module.title }}
-                                    </h3>
-                                    {{ learning_module.heading | richtext| add_govuk_classes }}
-                                </div>
-                                <div class="progess-container great-flex-1">
-                                    {% if module_completion_data.completion_count %}
-                                        {% include "learn/_high_level_progress.html" with module_completion_data=module_completion_data only %}
-                                    {% endif %}
-                                </div>
-                            </div>
-                        </div>
-                    </a>
+                    {% get_page_url learning_module as learning_module_url %}
+                    {% render_automated_list_page_card_content learning_module request module_completion_data as automated_page_content %}
+                    {% include 'components/great/card.html' with url=learning_module_url content=automated_page_content bypass_class_filter=True title=learning_module.title show_title_link=True heading_class="govuk-heading-s" classes="great-card govuk-!-margin-bottom-4" heading_level=h2 %}
```

Comment:
You can remove the heading_class because `-s` is the default class if the attribute isn't set. 
You can also remove the `great-card` class, as that's already set on the component. You'll need to remove the margin class because that's preventing the "pressed down" active working properly.
The extra preceding space may be the reason why `heading_level` isn't working.
```suggestion
                    {% include 'components/great/card.html' with url=learning_module_url content=automated_page_content bypass_class_filter=True title=learning_module.title show_title_link=True heading_level=h2 %}
```